### PR TITLE
Chapter on mass distribution.

### PR DESCRIPTION
**Pull request by moorepants**

### CODE REVIEW

**Comment by moorepants** [comment]
This still a little rough and I don't have much code examples. I may also add the parallel axis theorem.

File: mass.rst [hunk 1]
```diff
@@ -0,0 +1,504 @@
...
+Inertia Matrix
+==============
+
+For mutually perpendicular unit vectors fixed in reference frame :math:`A`, the
+moments of inertia with respect to :math:`O` about each unit vector and the
+products of inertia among the pairs of perpendicular unit vectors can also be
+computed. This, in general, results in nine inertia scalars that describe the
+mass distribution of a set of particles or a solid body in 3D space. These
+scalars are typically presented as a symmetric *inertia matrix* (also called an
+*inertia tensor*) that takes this form:
```
Comment (jtheinen) [inline mass.rst:202]: Maybe add that the moments of inertia are on the diagonal and the products of inertia are the rest, to support the explanation of the definitions given in line 170.

File: mass.rst [hunk 2]
```diff
@@ -0,0 +1,504 @@
...
+
+   me.outer(A.y, A.z)
+
+.. jupyter-execute::
+
+   me.outer(A.y, A.z).to_matrix(A)
+
+These unit dyads can be formed from unit vectors that are fixed in different
+reference frames. This is convenient because we can create dyadics, just like
+vectors, which are make up of components in different reference frames. For
```
Comment (jtheinen) [inline mass.rst:311]: make = made

Comment (jtheinen) [reply mass.rst:311]: sentence in 309 says the same as sentence in 310

File: mass.rst [hunk 3]
```diff
@@ -0,0 +1,504 @@
...
+.. math::
+
+   \bar{I}_a = \hat{n}_a \cdot \breve{I}
+
+.. todo:: Add the rigid body form of the inertia dyadic equation.
+
+Note that we have now described the inertia of the set of particles without
+needing to specify a vector :math:`\hat{n}_a`. The vectors and dyadics in Eq.
+:math:numref:`eq-inertia-dyadic` can be written in terms of any reference frame
+unit vectors of unit dyads.
```
Comment (jtheinen) [inline mass.rst:454]: what are reference frame unit vectors of unit dyads?

**Review by jtheinen — COMMENTED** [review]
This chapter is a bit harder to go through. It would make everything a lot easier if I'd know why and what for I can use the formulas. Probably if you add some examples it will be a lot easier to comprehend

File: mass.rst [hunk 4]
```diff
@@ -0,0 +1,504 @@
...
+   :label: eq-zeroth-moment-rigid-body
+
+   m := \int_{\textrm{solid}} \rho dV
+
+Mass Center
+===========
+
+If each particle in a set of particles is located at positions
+:math:`\bar{r}_i,\ldots,\bar{r}_\nu` relative to a point :math:`O` and the
+first mass moment, :math:`\sum_{i=1}^\nu m_i \bar{r}_i` is equal to 0 then the
```
[the remark this answers is not in the document]
Comment (moorepants) [reply mass.rst]: Adjusted

File: mass.rst [hunk 5]
```diff
@@ -0,0 +1,504 @@
...
+
+   r_O_So.xreplace({m2: 2*m1, m3: 3*m1}).simplify()
+
+Mass Distribution
+=================
+
+The inertia, or second moment of mass, describes the distribution of mass
+relative to a point about an axis. For a set of particles
+:math:`P_1,\ldots,P_\nu` with positions :math:`\bar{r}^{P_i/O}` for
+:math:`i=1,\ldots,\nu` relative to a point :math:`O` the *inertia vector* about
```
[the remark this answers is not in the document]
Comment (moorepants) [reply mass.rst]: added

File: mass.rst [hunk 6]
```diff
@@ -0,0 +1,504 @@
...
+If :math:`m_2=2m_1` and :math:`m_3=3m_1` then:
+
+.. jupyter-execute::
+
+   r_O_So.xreplace({m2: 2*m1, m3: 3*m1}).simplify()
+
+Mass Distribution
+=================
+
+The inertia, or second moment of mass, describes the distribution of mass
```
[the remark this answers is not in the document]
Comment (moorepants) [reply mass.rst:104]: Added a sentence.

File: mass.rst [hunk 7]
```diff
@@ -0,0 +1,504 @@
...
+reduces to:
+
+.. math::
+   :label: eq-moment-of-inertia
+
+   I_{aa} =
+   \sum_{i=1}^\nu m_i
+   \left( \bar{r}^{P_i/O} \times \hat{n}_a \right)^2
+
+The *radius of gyration* about a line through :math:`O` parallel to
```
[the remark this answers is not in the document]
Comment (moorepants) [reply mass.rst:198]: improved

File: mass.rst (hunk 1 above)
Comment (moorepants) [reply mass.rst:214]: done

File: mass.rst (hunk 2 above)
Comment (moorepants) [reply mass.rst:323]: fixed

File: mass.rst (hunk 3 above)
Comment (moorepants) [reply mass.rst:466]: unit dyads are the unit vectors of 2nd order tensors, i tried to clarify

Comment (moorepants) [reply mass.rst:466]: added "respectively"

**Pull request merged**